app/views/users/edit.html.erb

```diff
@@ -28,7 +28,18 @@
   <% @schedules.each do |schedule| %>
     <tr>
       <td><span class="glyphicon glyphicon-calendar" aria-hidden="true"></span> <%= "#{schedule.title}" %></td>
-      <td><%= link_to edit_user_schedule_path(@guider, schedule), title: "Edit Schedule #{schedule.title}", class: 'btn btn-info' do %><span class="glyphicon glyphicon-pencil" aria-hidden="true"></span><span class="sr-only">Edit Schedule</span><% end %></td>
+      <td nowrap="true">
+        <%= link_to edit_user_schedule_path(@guider, schedule), title: "Edit Schedule #{schedule.title}", class: 'btn btn-info' do %><span class="glyphicon glyphicon-pencil" aria-hidden="true"></span><span class="sr-only">Edit Schedule</span><% end %>
+        <% if schedule.destroyable? %>
+          <%= link_to user_schedule_path(@guider, schedule), method: :delete, title: 'Delete Schedule', class: 'btn btn-danger t-delete' do %>
+            <span class="glyphicon glyphicon-remove" aria-hidden="true"></span><span class="sr-only">Delete Schedule</span>
+          <% end %>
+        <% else %>
+          <a href='javascript:void(0)' disabled title='Delete Schedule', class='btn btn-danger'>
```

Comment:
Ideally both delete buttons should be a `<button>`, as semantically we're performing an action rather than navigating to a new page. Browsers also honour the `disabled` attribute on those elements so we can avoid the href weirdness.